Add Repository Sets as a new procedure (#1593)
theforeman/foreman-documentation: guides/common/modules/proc_changing-the-repository-sets.adoc

> [id="Changing_the_Repository_Sets_Status_in_Project{context}"]

Reviewer: ```suggestion
[id="Changing_the_Repository_Sets_Status_in_Project_{context}"]
```

= Changing the Repository Sets Status in {Project}

Repository sets show available repositories to each host.
If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts*, and select the host whose repository sets you want to change.

> . Select a host and select the *Repository sets* tab.
> On the *Repository sets* tab, there is a set of available repositories available to each host with a status of *Enabled* or *Disabled*.

Reviewer: ```suggestion
. Select the *Repository sets* tab.
On the tab, there is a set of repositories available to each host with a status of *Enabled* or *Disabled*.
```

. Select *Override to disabled*, *Override to enabled*, or *Reset to default* from the action menus on each row to overwrite the default status.
. Optional: Verify the environment for the host in *Content view details* in the *Overview* tab.
+
[NOTE]
====

> For hosts not in a default view or a lifecycle environment, the repository tab looks different.

Reviewer: Does this refer to the **Repository sets** tab? If yes, I think it should be specified.

> For hosts not in a default view or a lifecycle environment, the repository tab looks different.
> It defaults to showing only the repositories that are relevant to the host.

Reviewer: ```suggestion
For hosts not in a default view or a lifecycle environment, the repository tab shows only repositories that are relevant to the host.
```
